utils/create_dummy_models.py

```diff
@@ -248,12 +265,16 @@ def build_processor(config_class, processor_class):
 
     # validation
     if processor is not None:
-        assert isinstance(processor, processor_class) or processor_class.__name__.startswith("Auto")
+        if not (isinstance(processor, processor_class) or processor_class.__name__.startswith("Auto")):
+            raise ValueError(
+                f"`processor` (which is of type {processor.__class__.__name__}) should be an instance of"
+                f" {processor_class.__name__} or an Auto class!"
+            )
 
     return processor
 
 
-def get_tiny_config(config_class):
+def get_tiny_config(config_class, **model_tester_kwargs):
```

Comment:
To pass some arguments to model tester class's `__init__`, for example, `vocab_size`, `xxx_token_id` etc. See the explanation in `get_config_overrides`.